src/main/resources/twitter-demo/run_tweetbook_demo.py

```diff
@@ -23,7 +23,7 @@
 
 import sys
 if len(sys.argv)==1:
-    asterix_server = "http://localhost:19002"
+    asterix_server = "http://kiwi.ics.uci.edu:19002"
```

Comment:
should not change the default one. 
To use the kiwi instance you can run the command like this
```
python run_tweetbook_demo.py http://kiwi.ics.uci.edu:19002
```